Sturm documentation (#3971)
ESCOMP/CTSM: doc/source/tech_note/Soil_Snow_Temperatures/CLM50_Tech_Note_Soil_Snow_Temperatures.rst

Reviewer: Since the equation labels have changed above, I don't think these are the right equation numbers anymore, which would be an arguement for using a word label for the new equation you added.

Reviewer: I think this should now be :eq:6.90 and :eq:6.93

Reviewer: Sorry, I finally was able to look at the build of your changes and it looks like it would be 6.88 and 6.91.
The first equation is after "depends on the heat capacities of the soil solid, liquid water, and ice constituents", and the second equation is after "is the heat capacity of organic matter. For glaciers and snow".  Sorry, this is a dumb way to suggest changes.

Reviewer: But no, what counts are the labels, and those two equations are now labeled 6.90 and 6.93, even though they resolve as 6.88 and 6.91.  So my first suggestion should be correct (6.90 and 6.93).

Author: where :math:`c_{1}^{*}`  is calculated from :eq:`6.90` or :eq:`6.93`.

Reviewer: Correct.

Author: ```suggestion
where :math:`c_{1}^{*}`  is calculated from :eq:`6.88` or :eq:`6.91`.
```

.. math::
   :label: 6.87

   \lambda _{i} = \left\{
   \begin{array}{lr}
   0.023 + 0.234(\rho _{sno,\, i} /1000) &\qquad \rho _{sno,\, i} <= 156 \\
   0.138 - 1.01(\rho _{sno,\, i} /1000) + 3.233(\rho _{sno,\, i} /1000)^2 &\qquad \rho _{sno,\, i} > 156
   \end{array}\right\} 

with the bulk density of snow :math:`\rho _{sno,\, i}` (kg m\ :sup:`-3`) calculated as the mass of ice and liquid water per unit volume of snow layer 

.. math::
   :label: 6.88

   \rho _{sno,\, i} =\frac{w_{ice,\, i} +w_{liq,\, i} }{\Delta z_{i} } .
   
Previous versions of CLM used the :ref:`Jordan (1991) <Jordan1991>` parameterization for snow thermal conductivity, which uses the thermal conductivity of air and the bulk density of snow as: 

.. math::
   :label: 6.89

   \lambda _{i} =\lambda _{air} +\left(7.75\times 10^{-5} \rho _{sno,\, i} +1.105\times 10^{-6} \rho _{sno,\, i}^{2} \right)\left(\lambda _{ice} -\lambda _{air} \right)

where :math:`\lambda _{air}` is the thermal conductivity of air (:numref:`Table Physical Constants`) and :math:`\rho _{sno,\, i}` calculated, as in :eq:`6.88`.

> The Sturm (1997) parameterization is used in CLM6 over vegetated, glacier, and lake land units, but users can choose to use the Jordan (1991) parameterization over any of these land units. This can be accomplished through a change to the lnd_in namelist by using user_nl_clm.

Author: ```suggestion
The :ref:`Sturm et al. (1997)<Sturmetal1997>` parameterization is used in CLM6 over vegetated, glacier, and lake land units, but users can choose to use the Jordan (1991) parameterization over any of these land units. This can be accomplished through a change to the lnd_in namelist by using user_nl_clm.
```

   S_{r,\, i} =\left(\frac{w_{liq,\, i} }{\rho _{liq} \Delta z_{i} } +\frac{w_{ice,\, i} }{\rho _{ice} \Delta z_{i} } \right)\frac{1}{\theta _{sat,\, i} } =\frac{\theta _{liq,\, i} +\theta _{ice,\, i} }{\theta _{sat,\, i} } \le 1.

> 

Reviewer: Is there an extra blank line here?

> 

Author: ```suggestion
```

> Thermal conductivity :math:`\lambda _{i}` (W m\ :sup:`-1` K\ :sup:`-1`) for snow in CLM6.0 is modified from :ref:`Sturm et al. (1997)<Sturmetal1997>`, as applied by :ref:`Dutch et al. (2022)<Dutchetal2022>` and :ref:`Damseaux et al. (2025)<Damseauxetal2025>`. The Sturm function uses of the bulk density of snow to determine the thermal conductivity of snow layer (:math:`_{i}`) as:

Reviewer: Change "uses of the bulk density" to "uses the bulk density"

> Thermal conductivity :math:`\lambda _{i}` (W m\ :sup:`-1` K\ :sup:`-1`) for snow in CLM6.0 is modified from :ref:`Sturm et al. (1997)<Sturmetal1997>`, as applied by :ref:`Dutch et al. (2022)<Dutchetal2022>` and :ref:`Damseaux et al. (2025)<Damseauxetal2025>`. The Sturm function uses of the bulk density of snow to determine the thermal conductivity of snow layer (:math:`_{i}`) as:

Author: ```suggestion
Thermal conductivity :math:`\lambda _{i}` (W m\ :sup:`-1` K\ :sup:`-1`) for snow in CLM6.0 is modified from :ref:`Sturm et al. (1997)<Sturmetal1997>`, as applied by :ref:`Dutch et al. (2022)<Dutchetal2022>` and :ref:`Damseaux et al. (2025)<Damseauxetal2025>`. The Sturm function uses the bulk density of snow to determine the thermal conductivity of snow layer (:math:`_{i}`) as:
```

.. math::
   :label: 6.87

   \lambda _{i} = \left\{
   \begin{array}{lr}
   0.023 + 0.234(\rho _{sno,\, i} /1000) &\qquad \rho _{sno,\, i} <= 156 \\

>    0.138 - 1.01(\rho _{sno,\, i} /1000) + 3.233((\rho _{sno,\, i} /1000)^2) &\qquad \rho _{sno,\, i} > 156

Reviewer: This second term here seems to render with an extra set of parentheses. So instead:
$3.233(\rho _{sno,\, i} /1000)^2$

>    0.138 - 1.01(\rho _{sno,\, i} /1000) + 3.233((\rho _{sno,\, i} /1000)^2) &\qquad \rho _{sno,\, i} > 156

Author: ```suggestion
   0.138 - 1.01(\rho _{sno,\, i} /1000) + 3.233(\rho _{sno,\, i} /1000)^2 &\qquad \rho _{sno,\, i} > 156
```

   \end{array}\right\} 

> With the bulk density of snow :math:`\rho _{sno,\, i}` (kg m\ :sup:`-3`) calculated as the mass of ice and liquid water per unit volume of snow layer 

Reviewer: Change "With" to "with" (continuation of sentence above?).

> With the bulk density of snow :math:`\rho _{sno,\, i}` (kg m\ :sup:`-3`) calculated as the mass of ice and liquid water per unit volume of snow layer 

Author: ```suggestion
with the bulk density of snow :math:`\rho _{sno,\, i}` (kg m\ :sup:`-3`) calculated as the mass of ice and liquid water per unit volume of snow layer 
```

.. math::
   :label: 6.88

   \rho _{sno,\, i} =\frac{w_{ice,\, i} +w_{liq,\, i} }{\Delta z_{i} } .
   

> Previou versions of CLM used the :ref:`Jordan (1991) <Jordan1991>` parameterization for snow thermal conductivity, which uses the thermal conductivity of air and the bulk density of snow as: 

Reviewer: Change "Previou" to "Previous"

> Previou versions of CLM used the :ref:`Jordan (1991) <Jordan1991>` parameterization for snow thermal conductivity, which uses the thermal conductivity of air and the bulk density of snow as: 

Author: ```suggestion
Previous versions of CLM used the :ref:`Jordan (1991) <Jordan1991>` parameterization for snow thermal conductivity, which uses the thermal conductivity of air and the bulk density of snow as: 
```

> .. math::

Reviewer: Can we get around the equation numbering changes that follow this equation by using a word label here as @slevis-lmwg has suggested?

Author: Can you suggest what this would look like, @olyson? I'm not sure I follow

Reviewer: Actually, I guess that wouldn't work anyway because the equations will be renumbered when the document is built, so never mind.

   :label: 6.89

   \lambda _{i} =\lambda _{air} +\left(7.75\times 10^{-5} \rho _{sno,\, i} +1.105\times 10^{-6} \rho _{sno,\, i}^{2} \right)\left(\lambda _{ice} -\lambda _{air} \right)

where :math:`\lambda _{air}` is the thermal conductivity of air (:numref:`Table Physical Constants`) and :math:`\rho _{sno,\, i}` calculated, as in :eq:`6.88`.

> The Sturm (1997) parameterization is used in CLM6 over vegetated, glacier, and lake land units, but users can choose to use the Jordan (1991) parameterization over any of these land units if desired.

Reviewer: Maybe mention that this can be accomplished through a change to the lnd_in namelist by using user_nl_clm.

> The Sturm (1997) parameterization is used in CLM6 over vegetated, glacier, and lake land units, but users can choose to use the Jordan (1991) parameterization over any of these land units if desired.

Reviewer: Use the clickable versions of the references here for Sturm (1997) and Jordan (1991)?  If not, at least Sturm (1997) should be Sturm et al. (1997).

> The Sturm (1997) parameterization is used in CLM6 over vegetated, glacier, and lake land units, but users can choose to use the Jordan (1991) parameterization over any of these land units if desired.

Author: ```suggestion
The Sturm (1997) parameterization is used in CLM6 over vegetated, glacier, and lake land units, but users can choose to use the Jordan (1991) parameterization over any of these land units. This can be accomplished through a change to the lnd_in namelist by using user_nl_clm.
```

where :math:`c_{s,bedrock} =2\times 10^{6}` J m\ :sup:`-3` K\ :sup:`-1` is the heat capacity of bedrock and :math:`c_{s,om} =2.5\times 10^{6}` \ J m\ :sup:`-3` K\ :sup:`-1` (:ref:`Farouki 1981 <Farouki1981>`) is the heat capacity of organic matter. For glaciers and snow

.. math::
   :label: 6.93

   c_{i} =\frac{w_{ice,\, i} }{\Delta z_{i} } C_{ice} +\frac{w_{liq,\, i} }{\Delta z_{i} } C_{liq} .

For the special case when snow is present (:math:`W_{sno} >0`) but there are no explicit snow layers (:math:`snl=0`), the heat capacity of the top layer is a blend of ice and soil heat capacity

.. math::
   :label: 6.94

   c_{1} =c_{1}^{*} +\frac{C_{ice} W_{sno} }{\Delta z_{1} }

> where :math:`c_{1}^{*}`  is calculated from :eq:`6.88` or :eq:`6.91`.

Author: ```suggestion
where :math:`c_{1}^{*}`  is calculated from :eq:`6.89` or :eq:`6.91`.
```

> where :math:`c_{1}^{*}`  is calculated from :eq:`6.89` or :eq:`6.91`.

Author: ```suggestion
where :math:`c_{1}^{*}`  is calculated from :eq:`6.88` or :eq:`6.91`.
```

> where :math:`c_{1}^{*}`  is calculated from :eq:`6.88` or :eq:`6.91`.

Author: @samsrabin can you help us point to the correct equations here?

Reviewer: @wwieder , I was able to suggest a change and apply it.  I then built it and it looks like we are pointing to the correct equations now.

> where :math:`c_{1}^{*}`  is calculated from :eq:`6.88` or :eq:`6.91`.

Reviewer: ```suggestion
where :math:`c_{1}^{*}`  is calculated from :eq:`6.90` or :eq:`6.93`.
```
